### PR TITLE
Prince feedback component

### PR DESCRIPTION
**Pull request by princegeorge121**

### CODE REVIEW

File: client/mycart/src/app/customer/customer-routing.module.ts [hunk 1]
```diff
@@ -3,12 +3,14 @@ import { RouterModule, Routes } from '@angular/router';
...
 import { CustomerHomeComponent } from './customer-home/customer-home.component';
 import { CustomerLayoutComponent } from './customer-layout/customer-layout.component';
+import { FeedbackComponent } from './feedback/feedback.component';
 
 const routes: Routes = [
   {
     path: '', component: CustomerLayoutComponent, children: [
         { path: 'cart', component: CartComponent },
-        { path: 'customerHome', component: CustomerHomeComponent }
+        { path: 'customerHome', component: CustomerHomeComponent },
```
Comment (mishelshaji) [inline client/mycart/src/app/customer/customer-routing.module.ts]: Change the URL to '' or 'home'.

File: client/mycart/src/app/customer/feedback/feedback.component.html [hunk 2]
```diff
@@ -1 +1,45 @@
...
+                    #feedback="ngModel" required></textarea>
+                <div class="text-danger" *ngIf="feedback.invalid && feedback.touched">
+                    <div *ngIf="feedback.errors?.['minlength']">The given feedbackis too small.</div>
+                    <div *ngIf="feedback.errors?.['maxlength']">The given geedback is too long.</div>
+                    <div *ngIf="feedback.errors?.['required']">Feedback is required.</div>
+                </div>
+            </div>
+            <div class="d-flex justify-content-center align-items-center mt-5 mb-5">
+                <button type="submit" class="btn btn-primary"
+                    [disabled]="email.invalid && fullname.invalid && feedback.invalid">Submit</button>
```
Comment (mishelshaji) [inline client/mycart/src/app/customer/feedback/feedback.component.html]: Use form model's invalid property to disable the button.

File: client/mycart/src/app/customer/feedback/feedback.component.ts [hunk 3]
```diff
@@ -6,5 +6,13 @@ import { Component } from '@angular/core';
   styleUrls: ['./feedback.component.css']
 })
 export class FeedbackComponent {
+  model = {
+    email: '',
+    fullname: '',
+    feedback:''
```
Comment (mishelshaji) [inline client/mycart/src/app/customer/feedback/feedback.component.ts]: Format the code.

File: client/mycart/src/app/customer/feedback/feedback.component.ts [hunk 4]
```diff
@@ -6,5 +6,13 @@ import { Component } from '@angular/core';
...
+  };
+  onSubmit(form: any) {
```
Comment (mishelshaji) [inline client/mycart/src/app/customer/feedback/feedback.component.ts:16]: Add a blank line above.

**Review by mishelshaji — CHANGES_REQUESTED** [review]

File: client/mycart/src/app/customer/feedback/feedback.component.ts (hunk 4 above)
Comment (princegeorge121) [reply client/mycart/src/app/customer/feedback/feedback.component.ts:16]: Done

File: client/mycart/src/app/customer/feedback/feedback.component.html (hunk 2 above)
Comment (princegeorge121) [reply client/mycart/src/app/customer/feedback/feedback.component.html]: Done.

File: client/mycart/src/app/customer/feedback/feedback.component.ts (hunk 3 above)
Comment (princegeorge121) [reply client/mycart/src/app/customer/feedback/feedback.component.ts]: Done.

File: client/mycart/src/app/customer/customer-routing.module.ts (hunk 1 above)
Comment (princegeorge121) [reply client/mycart/src/app/customer/customer-routing.module.ts]: Done

**Review by mishelshaji — APPROVED** [review]

**Pull request merged**